doc/schemas/close.request.json

```
@@ -23,7 +23,7 @@
       "description": ""
     },
     "wrong_funding": {
-      "type": "txid",
+      "type": "outpoint",
```

```suggestion
      "type": "txid",
```

I think this is the compiler error problem, sorry but I did not pay too much attention to the error message :/

There is any reason to change into `output` that I'm missing?

P.S: Code generator are lovely but too magic :)

---

Yes, it's actually outpoint, not txid, see manpage.

---

you have right! so at this point I think @cdecker is the best person to make rust happy here

---

Yeah, sadly this is also a breaking change for the rust bindings. Though we don't yet have a compatibility guarantee, so we can break on a whim imho.